apps/omg_eth/test/support/dev_geth.ex

```diff
@@ -103,7 +103,7 @@ defmodule OMG.Eth.DevGeth do
 
     waiting_task
     |> Task.async()
-    |> Task.await(30_000)
+    |> Task.await(90_000)
```

Comment:
just curious what is leading to this change?

Reply:
dev geth is timing out which causes ci failures. I increased it so dev geth has more time to prepare itself